.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,43 @@
 <!-- Please review the following before submitting a PR:
 Che's Contributing Guide: https://github.com/eclipse/che/blob/master/CONTRIBUTING.md
 Pull Request Policy: https://github.com/eclipse/che/wiki/Development-Workflow#pull-requests
-
-COMMITTERS: please include labels on each PR. Labels are listed here: https://github.com/eclipse/che/wiki/Labels but at a minimum you should include `kind/..` and `Dev Open Pull Request Status` labels.
 -->
 
 ### What does this PR do?
 
 
+### Screenshot/screencast of this PR
+<!-- Please include a screenshot or a screencast explaining what is doing this PR -->
+
+
 ### What issues does this PR fix or reference?
+<!-- Please include any related issue from eclipse che repository (or from another issue tracker).
+     Include link to other pull requests like documentation PR from [the docs repo](https://github.com/eclipse/che-docs)
+-->
+
+
+### How to test this PR?
+<!-- Please explain for example :
+  - The test platform (openshift, kubernetes, minikube, CodeReady Container, docker-desktop, etc)
+  - Installation method: chectl / che-operator
+  - steps to reproduce
+ -->
+
 
-<!-- #### Changelog -->
-<!-- The changelog will be pulled from the PR's title. 
-     Please provide a clear and meaningful title to the PR and don't include issue number -->
+### PR Checklist
 
+As the author of this Pull Request I made sure that:
 
-#### Release Notes
-<!-- markdown to be included in marketing announcement - N/A for bugs -->
+- [ ] The Eclipse Contributor Agreement is valid
+- [ ] Code produced is complete
+- [ ] Code builds without errors
+- [ ] Add new automated test for every bug fixed
+- [ ] The repository devfile is up to date and works
```

Comment:
what is "The repository devfile"?

Reply:
the devfile stored in this repository to be able to use dogfooding ?

Reply:
I think it's about the devfile, which is in the repository root. It should be aligned to changes in PR.

Reply:
got it. Most of the PRs won't update that devfile, but good to remind author. +1